pkg/query-service/app/logs/v3/query_builder.go

```diff
@@ -99,8 +72,12 @@ func getClickhouseLogsColumnDataType(columnDataType v3.AttributeKeyDataType) str
 }
 
 // getClickhouseColumnName returns the corresponding clickhouse column name for the given attribute/resource key
-func getClickhouseColumnName(key v3.AttributeKey, fields map[string]v3.AttributeKey) (string, error) {
+func getClickhouseColumnName(key v3.AttributeKey) (string, error) {
 	clickhouseColumn := key.Key
+	if key.Key == "timestamp" || key.Key == "id" {
+		return key.Key, nil
+	}
```

Comment:
Do you need this conditional block here? Since both are top-level columns, `return clickhouseColumn, nil` below does the same?

Reply:
The reason for this is that in the `attributes` api we don't return `timestamp` and `id` . So the frontend won't have any context 

ex:-
```
"A": {
                "queryName": "A",
                "dataSource": "logs",
                "aggregateOperator": "noop",
                "expression": "A",
                "disabled": false,
                "pageSize": 10,
                "filters": {
                    "items": [
                        {
                            "key": {
                                "key": "method",
                                "type": "tag",
                                "dataType": "string"
                            },
                            "op": "=",
                            "value": "GET"
                        }
                    ],
                    "op": "AND"
                },
                "orderBy": [
                    {
                        "columnName": "timestamp",
                        "order": "desc"
                    }
                ]
            }
   ```
   
   Here we don't need to enrich.